app/infrastructure/scrapers/kutis_scraper.py

```
@@ -0,0 +1,136 @@
+from app.infrastructure.constants import (
+    KUTIS_TIMETABLE_PAGE_URL,
+)
+from app.infrastructure.exceptions import ErrorType, ScraperException
+
+
+class KutisScraper:
+    def __init__(self, client: KutisClient):
+        self.client = client
+
+    async def _fetch(
+        self, method: HTTPMethod, url: str, data: dict | None = None
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)